Tests/SnapshotConfiguration.swift

```diff
@@ -70,6 +73,9 @@ extension SnapshotConfiguration {
     // Test cases for `AnimatedImageProvider`
     "Nonanimating/_dog": .customImageProvider(HardcodedImageProvider(imageName: "Samples/Images/dog.png")),
 
+    // Test cases for `AnimatedTextProvider`
+    "Issues/issue_1722": .customTextProvider(HardcodedTextProvider(text: "Bounce-bounce")),
```

Comment:
When you're done with the code changes you can generate the snapshot images for this by running the unit test target on an iPhone 8 simulator (or by running `bundle exec rake test:package` on the command line).

Reply:
fyi I think the snapshot tests are sensitive to specific Xcode versions -- right now I think they run correctly on Xcode 13.4.1 but not on Xcode 14. If you get a lot of spurious failures when you run the tests let me know and I can pull you branch and run them for you.

Reply:
Hm unfortunately it always return this message during the test run: 
`No reference was found on disk. Automatically recorded snapshot: `

Reply:
> If you get a lot of spurious failures when you run the tests let me know and I can pull you branch and run them for you.

Yeah, Cal, that would be helpful, thanks a bunch! 🙏🏼

Reply:
By the way the name of the text layer in the provided animation is `Animation text layer` 🙏🏼

Reply:
> Hm unfortunately it always return this message during the test run:
> No reference was found on disk. Automatically recorded snapshot:

This is expected the first time you run the tests after you add a new sample JSON file. If you run the tests again this goes away, since now it compares the new snapshot with the previously-recorded one.

Reply:
Hm true. However, the snapshots anyway don't contain the text. Could it be because the text layer has a different name?

Reply:
I think this was because we had to call `textRenderLayer.sizeToFit()` after updating `textRenderLayer.text`.